src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -56,18 +56,6 @@ public class Providers {
     // Note volatile immutable object, so no synchronization needed.
     private static volatile ProviderList providerList;
 
-    static {
-        // set providerList to empty list first in case initialization somehow
-        // triggers a getInstance() call (although that should not happen)
-        providerList = ProviderList.EMPTY;
-        providerList = ProviderList.fromSecurityProperties();
-        RestrictedSecurity.checkHashValues();
-    }
-
-    private Providers() {
-        // empty
-    }
```

Comment:
Please leave the constructor where it was.

Reply:
Hi Keith. I don't believe that I moved the constructor here, instead i just moved `restrictedJarVerificationProviders` and `jarVerificationProviders` above the static block. The constructor `Providers()` is right after the `static {}` both before and after the update. Perhaps its the way that github is lining up the dif here and showing movement of the static block and constructor instead of showing movement of the `restrictedJarVerificationProviders` and `jarVerificationProviders` values? In any case I can move the `Providers()` constructor above both
`restrictedJarVerificationProviders` and `jarVerificationProviders` also if that is what we want to do?

Reply:
In the [latest version of this file from upstream](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/5dd840a87ed8d325e0a690e901bc2c385438b8eb/src/java.base/share/classes/sun/security/jca/Providers.java), the declaration of `providerList` is followed by the `static` block (that needs to move), then the constructor. After this change, the constructor should immediately follow the declaration of `providerList`.

Reply:
OK, moved constructor to after `providerList` declaration using commit [de10fad](https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/844/commits/de10fad3038f8f28995a263e9be8ccfe97cfc7df). The dif now represents moving the static block down below the `restrictedJarVerificationProviders` and `jarVerificationProviders` values instead of the reverse.